content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -105,30 +105,17 @@ If you do not specify either, then the DaemonSet controller will create Pods on
 
 ## How Daemon Pods are scheduled
 
-### Scheduled by default scheduler
-
-{{< feature-state for_k8s_version="1.17" state="stable" >}}
-
-A DaemonSet ensures that all eligible nodes run a copy of a Pod. Normally, the
-node that a Pod runs on is selected by the Kubernetes scheduler. However,
-DaemonSet pods are created and scheduled by the DaemonSet controller instead.
-That introduces the following issues:
-
-* Inconsistent Pod behavior: Normal Pods waiting to be scheduled are created
-  and in `Pending` state, but DaemonSet pods are not created in `Pending`
-  state. This is confusing to the user.
-* [Pod preemption](/docs/concepts/scheduling-eviction/pod-priority-preemption/)
-  is handled by default scheduler. When preemption is enabled, the DaemonSet controller
-  will make scheduling decisions without considering pod priority and preemption.
-
-`ScheduleDaemonSetPods` allows you to schedule DaemonSets using the default
-scheduler instead of the DaemonSet controller, by adding the `NodeAffinity` term
-to the DaemonSet pods, instead of the `.spec.nodeName` term. The default
-scheduler is then used to bind the pod to the target host. If node affinity of
-the DaemonSet pod already exists, it is replaced (the original node affinity was
-taken into account before selecting the target host). The DaemonSet controller only
-performs these operations when creating or modifying DaemonSet pods, and no
-changes are made to the `spec.template` of the DaemonSet.
+A DaemonSet ensures that all eligible nodes run a copy of a Pod. The DaemonSet
+controller creates a Pod for each eligible node and adds the
+`spec.affinity.nodeAffinity` field of the Pod to match the target host. After
+the Pod creation, the default scheduler takes over and binds the Pod to the
+target host by setting the `.spec.nodeName` field.
```

Comment:
I'd suggest some different wording, if we want to add this info:

```
target host by setting the `.spec.nodeName` field. If the new Pod cannot fit on the
node due to lack of resources, the control plane may evict some of the existing Pods
based on the [priority](/docs/concepts/scheduling-eviction/pod-priority-preemption/#pod-priority)
of the new Pod.
```

Reply:
Good point, the removal to make a more-important incoming pod fit is better termed _preemption_. I sometimes call it eviction when I'm not being strict about meanings.

Just to make things even less easy to remember, the mechanism to make preemption work is to create the `eviction` subresource of the targeted Pod&hellip;

You're right @gregth, it might not be resource pressure that drives preemption.

Reply:
@sftim Thanks for confirming. I just pushed another fixup commit to use the verb "preempt" and have evict in parenthesis.